snirf_specification.md

```diff
@@ -537,6 +537,119 @@ As described below, optional variables `probe.sourceLabels` and
 `probe.detectorLabels` are provided for indicating the instrument specific 
 label for sources and detectors.
 
+#### /nirs(i)/data(j)/measurementLists
+* **Presence**: optional
```

Comment:
Should this be 'required if `measurementList` is not present' for parity with @dboas changes to measurementList(k)?